packages/angular_devkit/core/src/index.ts

```diff
@@ -23,3 +23,5 @@ export {
   logging,
   terminal,
 };
+
+export const ENABLE_DIFFERENTIAL_LOADING = false;
```

Comment:
This is wrong. core has nothing to do with core package. 

I suggest override in the builder.

Ie, change;
```
const differentialLoading = isDifferentialLoadingNeeded(
    root: getSystemPath(root),projectRoot, target || projectTs.ScriptTarget.ES5)
```

to 
```
// const differentialLoading = isDifferentialLoadingNeeded(
    root: getSystemPath(root),projectRoot, target || projectTs.ScriptTarget.ES5)
const differentialLoading = false;
```

And xdescribe the builder specs for the time being.